.env

```diff
@@ -1 +1 @@
-#DATABASE_URL=postgresql://postgres:password@localhost/solaceassignment
+DATABASE_URL=postgresql://postgres:password@localhost/solaceassignment
```

Comment:
some people prefer .env to be not included in the repo.. here I'm just following existing conventions